provider/resource_entity.go

```diff
@@ -325,21 +333,22 @@ func (r *resourceEntity) Delete(ctx context.Context, req resource.DeleteRequest,
 }
 
 func (r *resourceEntity) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
-	b := entityBase(r.client, "", req.ID)
-	b, err := b.Find(ctx)
-	if err != nil {
-		resp.Diagnostics.AddError("Unable to find entity model", err.Error())
-		return
-	}
-	if b == nil {
-		resp.Diagnostics.AddError("Entity not found", fmt.Sprintf("Entity '%s' not found", req.ID))
-		return
-	}
-
-	state, diags := entityModelFromBase(ctx, b)
-	if resp.Diagnostics.Append(diags...); diags.HasError() {
-		return
-	}
-
-	resp.Diagnostics.Append(resp.State.Set(ctx, &state)...)
+	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
```

Comment:
I think, this is unnecessary, according to docs: 
>When the Read method requires multiple attributes to refresh, you must write custom logic in the ImportState method. 

(otherwise just ImportStatePassthroughID could be passed)

Reply:
I tested this :)

Reply:
Sure, but this wouldn't allow to import an entity by title though, right?

Reply:
That's totally true, though that in case we want to use the generate command, we won't need to import by title anyway

Reply:
Yes, but I still think it's good to have this functionality for the old-fashioned imports :)

Reply:
agree with Dmitry on this, import by title is really helpful.